R/doeResponseSurfaceMethodology.R

```diff
@@ -1008,12 +1012,12 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
     AnovaTable$dependOn(      options = c("anova","modelTerms", "rsmBlocks",
                                           "rsmResponseVariables",
                                           "rsmVariables"))
-    AnovaTable$addColumnInfo( name = "names",     title = " ")
-    AnovaTable$addColumnInfo( name = "Df",       title = "DF")
-    AnovaTable$addColumnInfo( name = "Sum",      title = "Sum of Squares")
-    AnovaTable$addColumnInfo( name = "Mean",     title = "Mean of Squares")
-    AnovaTable$addColumnInfo( name = "FValue",   title = "F")
-    AnovaTable$addColumnInfo( name = "PValue",   title = "p")
+    AnovaTable$addColumnInfo( name = "names",    title = gettextf(" "))
```

Comment:
```suggestion
    AnovaTable$addColumnInfo( name = "names",    title = "")
```
No need to translate "", it's the same in every language (at least the ones that I know of).